src/modules/clear.js

```diff
@@ -1,10 +1,10 @@
-// export default class Clear {
-//   constructor() {
-//     this.todos = JSON.parse(localStorage.getItem('todos')) || [];
-//   }
+export default class Clear {
+  constructor() {
+    this.todos = JSON.parse(localStorage.getItem('todos')) || [];
+  }
 
-//   clearTodos() {
-//     const filtered = this.todos.filter((t) => !t.completed);
-//     localStorage.setItem('todos', JSON.stringify(filtered));
-//   }
-// }
+  clearTodos() {
+    const filtered = this.todos.filter((t) => !t.completed);
+    localStorage.setItem('todos', JSON.stringify(filtered));
+  }
```

Comment:
- [ ] The indexes of the todos are not correctly updated after clearing the completed to-do items. Please implement this change as it messes up the order of the items.
![image](https://user-images.githubusercontent.com/46839548/215250824-899d5152-e622-4c1e-b366-e8d76fe98aab.png)